.github/CODEOWNERS

```diff
@@ -63,7 +63,7 @@
 /specification/cdn/ @jorinmejia @yunhemsft @jessicl-ms @rrahulms @ChenglongLiu @Ptnan7
 
 # PRLabel: %Cognitive Services
-/specification/cognitiveservices/ @felixwa @yangyuan
+/specification/cognitiveservices/ @rkuthala @MattGal
```

Comment:
@MattGal you need to make your Azure org visibility public.

![image](https://github.com/user-attachments/assets/ba0c09a0-f5f6-4f9a-9360-5d31e01ea657)

Reply:
@weshaggard done.